apps/playground/src/app/components/ag-grid/edit-in-modal-grid/ag-grid-edit-modal.component.ts

```diff
@@ -75,14 +75,14 @@ export class SkyAgGridEditModalComponent implements OnInit {
             skyComponentProperties: {
               data: SKY_DEPARTMENTS,
               selectionChange: (change: SkyAutocompleteSelectionChange) => {
-                this.departmentSelectionChange(change, params.node);
+                this.departmentSelectionChange(change, params.node as RowNode);
```

Comment:
is this necessary? Could we change the param type for `departmentSelectionChange`?